packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2037,6 +2046,21 @@ class EditableText extends StatefulWidget {
 
   bool get _userSelectionEnabled => enableInteractiveSelection && (!readOnly || !obscureText);
 
+  /// The default value for [selectAllOnFocus].
+  static bool get defaultSelectAllOnFocus {
```

Comment:
This can stay private, right? It doesn't seem like API that's needed from outside this class.